gui/system/migrations/0017_auto_20180219_1615.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-02-19 16:15
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('system', '0009_auto_20171023_2159'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Haven't we already done this here? https://github.com/freenas/freenas/blob/master/gui/system/migrations/0011_auto_20180219_1615.py

Reply:
@themylogin we do, however we need the same in stable and that cant be out of order and to keep it in sync with master. Is there a better way to do this?